spec/system/web_mentions/web_mention_microformats_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+# For webmentions to work, we basically need two things:
+# A source of webmentions(i.e: article on FOREM)
+# A target to which webmentions will be sent (i.e article's canonical URL)
+# Since FOREM will be the source of webmentions, every article should contain Microformats
+# that allows the receiving website to parse various information from the source article
+# such as the comment's content, the author of the comment and their information(name, picture, etc..)
+# Ref: http://microformats.org/wiki/h-entry
```

Comment:
I am not a huge code comment fan but this was very insightful and informative! Thank you!